src/node/index.ts

```diff
@@ -94,11 +94,12 @@ export default function PluginInspect(options: Options = {}): Plugin {
   let config: ResolvedConfig
 
   type TransformMap = Record<string, TransformInfo[]>
+  type ResolveIdMap = Record<string, ResolveIdInfo[]>
 
   const transformMap: TransformMap = {}
   const transformMapSSR: TransformMap = {}
-  const idMap: Record<string, string> = {}
-  const idMapSSR: Record<string, string> = {}
+  const idMap: ResolveIdMap = {}
+  const idMapSSR: ResolveIdMap = {}
```

Comment:
I made this to a `string` => `string[]` since an `id` can be resolved multiple times and it would be nice to measure all of them.